posts/2023-02-08-OMB-new-requirements-for-federal-executive-agencies.md

```diff
@@ -0,0 +1,24 @@
+---
+title: New domain requirements for federal executive branch agencies
+description: New requirements for the registration and use of .gov domains in the federal government
+layout: layouts/post
+date: 2023-02-08
+author: Cameron Dixon
+excerpt: OMB issued new requirements for the registration and use of .gov domains for federal executive branch agencies.
+tags:
+  - posts
+  - Domain requests
+  - Domain requirements
+  - Federal government executive branch
+---
+
+The Office of Management and Budget (OMB) issued new requirements for the registration and use of .gov domains for federal executive branch agencies.  
+
+Requirements include:
+-  **Agencies must use government domains** (i.e., .gov or .mil) for all official communications, information, and services, except for third-party services operated by non-governmental entities on non-governmental domains that are needed to effectively interact with the public. 
+- Agencies must **continue to report use of non-.gov domains** [How do they do this](#)?
+- Domain name requests must have the approval of the agency’s Chief Information Officer or the head of the agency
+
+OMB reviews domain requests to ensure that agencies are following relevant policies and domain name requirements, to help avert or resolve potential domain-naming conflicts between agencies, and to reduce confusion arising from overuse of similar domain names and misaligment between a domain name and its intended use. In areas of domain name conflict, OMB will seek resolution with the agencies involved. OMB may deny a domain name request or a domain name renewal, or may request that an agency transfer ownership of an existing domain to another agency to remediate potential conflict or confusion. 
+
+Read the full [OMB Memorandum 23-10](https://www.whitehouse.gov/wp-content/uploads/2023/02/M-23-10-DOTGOV-Act-Guidance.pdf).
```

Comment:
Should this link have a `usa-link--external` class?

Reply:
Yes, thank you!!